packages/ui-kit-components/src/components/Branding/Logo/Logo.tsx

```diff
@@ -1,19 +1,24 @@
-import { useTheme } from '@mui/material/styles';
+import { useTheme } from '@mui/material';
 
-import { LogoLarge, LogoMaximized, LogoSmall } from '../raw-icons';
+import {
+  darkLogoColor,
+  lightLogoColor,
+  LogoLarge,
+  LogoMaximized,
+  LogoSmall,
+} from '../branding';
 
 interface Props {
   variant?: string;
 }
 
 const Logo: React.FC<Props> = (svgProps) => {
   const theme = useTheme();
-  const color = theme.palette.logo;
+  const color = theme.palette.mode === 'dark' ? lightLogoColor : darkLogoColor;
```

Comment:
Curious, what was wrong with the approach of having the color on the theme?

Reply:
I thought it would be better to have it separate from the `theme` package in case users didn't want to use it.  Though I've noticed that the `theme` package is still being installed with this package.  So the benefits of having the logo colors separate from the theme might be negligible.